x-pack/plugins/security_solution/public/common/components/event_details/table/field_value_cell.tsx

```diff
@@ -36,7 +36,7 @@ export const FieldValueCell = React.memo(
     values,
   }: FieldValueCellProps) => {
     return (
-      <div>
+      <div data-test-subj={`event-field-${eventId}-${data.field}`}>
```

Comment:
I noticed in the snapshots that a long id was being placed as a part of the data-test-subj value. I think that ideally we could refrain from using dynamic values such as eventId for the data-test-subj values as it adds additional complexity (if we can get away with it of course) - is there a way to keep it easily grabbable and static? if that's the case I think we could go for it.

Reply:
I need a way to identify a specific field [here](https://github.com/elastic/kibana/pull/113274/files#diff-a32f3eba6563e8989a1d985f36a3998dff471fa6998e05d00d7b0bf9fd3afda6R138). I think that I can remove the event id and the test would pass because we can't open more than one event details tab at the same time. Let me do that! 👍